backend/forgather/docs/빠른_참조_가이드.md

```
@@ -0,0 +1,334 @@
+
+### 사용 가능한 API 메서드
+
+| 메서드 | 사용법 | 인증 헤더 |
+|--------|-------|-----------|
+| `API.get(endpoint, params, auth)` | GET 요청 | Yes (기본값) |
+| `API.post(endpoint, data, auth)` | POST 요청 | Yes (기본값) |
+| `API.put(endpoint, data, auth)` | PUT 요청 | Yes (기본값) |
+| `API.delete(endpoint, auth)` | DELETE 요청 | Yes (기본값) |
+
+예시:
+```javascript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)